module-4/backend/week_three.md

```diff
@@ -5,15 +5,46 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is Node?
+  Node by definition is a Server Side Enviroment for JavaScript, written in vanilla JS.  I commonly mistake it for a server side language similar to Ruby but that would be incorrect as it just transforms a client-side based language into a server-side capable language?
+
 2. What is Express? What is Express similar to in the Ruby world?
+  Express is a JS loosely-structred, unopinionated(supposedly) framework responsible for handling the incoming HTTP request and responses coming from clients and/or servers.  Express in Ruby terms would be tightly associated with Sinatra due to its high maluability for different aspects
+  
+  
 3. How do we setup a route when creating an API with Node and Express? Please provide a code snippet.
+  after installing and require/loading the express app into our JS server File to handle the incoming HTTP req/responses we use expresses built in 'get', 'post', 'patch', 'put', 'delete' and other various methods to handle the request and then handle the outgoing response. 
+  
+  app.get('/', function(request, response) {
+  response.sendFile(path.join(__dirname+'/welcome.html'))
+})
+
 4. What do we use Knex for?
+  Knex is our Object Relational Map tool similar to Active Record for database queries.  Very raw methods to query DB's without the raw SQL strings alone but with SQL understanding, inputing raw sql is relatively as easy or easier than AR.
```

Comment:
i wouldn't say it's an ORM - it's just a SQL query builder for us to help us connect/query our database while building a node app 👍